cdap-common/src/main/java/co/cask/cdap/common/namespace/AbstractNamespaceClient.java

```diff
@@ -100,7 +99,7 @@ public void create(NamespaceMeta namespaceMeta)
     if (response.getResponseCode() == HttpURLConnection.HTTP_BAD_REQUEST) {
       throw new BadRequestException("Bad request: " + responseBody);
     }
-    throw new IOException("Cannot get create namespace. Reason: " + "getDetails(response)");
+    throw new IOException("Cannot get create namespace. Reason: " + response);
```

Comment:
"Cannot create namespace. Reason: ", though if you can add the namespace name, even better:
```
String.format("Cannot create namespace '%s'. Reason: ", namespaceMeta.getName())
```